### PR TITLE
Implement HTTP server and get the pod, namespace and workload breakdowns.

### PR DESCRIPTION
**Pull request by AmartC**
This PR implements an HTTP server that will return pod, namespace and workload breakdowns depending on the GET request sent to the server. Test with this image: amartyarancher/opni-insights-service-amartya

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+FROM base
+EXPOSE 80
+
+COPY --from=builder /opt/venv /opt/venv
+ENV PATH="/opt/venv/bin:$PATH"
+
+COPY ./opni-insights-service/app /app
+
+WORKDIR "app"
+CMD ["uvicorn", "main:app", "--host", "0.0.0.0", "--port", "80"]
```
Comment (dbason) [inline Dockerfile:27]: We should be using the `rancher/opni-python-base:3.8` image to share layers with the other images.  If required we can add more prerequisites to that base image.

**Review by dbason — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (AmartC) [reply Dockerfile:27]: Okay, updating that now. Is kubernetes v12.0.1 a part of this Docker image currently?

**Review by dbason — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+## Insights service
+
+* This repository builds the opni-insights image which sets up an HTTP server that will return the breakdown of log messages by pod, namespace and workload.
```
Comment (sanjay920) [inline README.md:3]: can you add some sample documentation on how one would call any of the api endpoints?

e.g.
```
curl --location --request GET 'localhost:8082/workload?start_ts=1631794584000&end_ts=1631855784000' \
--header 'Content-Type: application/json'
```

**Review by sanjay920 — CHANGES_REQUESTED** [review]
Code looks good for POC purposes. Some minor changes requested elsewhere.

* should not be including .idea/* files

File: rbac.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: jupyter-rb
```
Comment (sanjay920) [inline rbac.yaml:4]: this name should change

**Review by sanjay920 — CHANGES_REQUESTED** [review]

File: rbac.yaml (hunk 3 above)
Comment (AmartC) [reply rbac.yaml]: Addressed.

**Review by sanjay920 — APPROVED** [review]

**Pull request merged**